packages/muelu/test/scaling/MatvecKernelDriver.cpp

```diff
@@ -306,8 +306,8 @@ class CuSparse_SpmV_Pack<double,LocalOrdinal,GlobalOrdinal,Kokkos::Compat::Kokko
     cols   = reinterpret_cast<int*>(Acolind_cusparse.data());
     rowptr = reinterpret_cast<int*>(Arowptr_cusparse.data());
 
-    auto X_lcl = X.template getLocalView<device_type> ();
-    auto Y_lcl = Y.template getLocalView<device_type> ();
+    auto X_lcl = X.template getLocalViewDevice<device_type> ();
```

Comment:
Why not use this instead here: `getLocalViewDevice(Tpetra::Access::ReadWrite)` with no template parameters to make life easy : )

Reply:
Thanks, fixed.